### ISSUE TITLE
Provisioner output hard to read due to packer filtering out blank lines

### ISSUE THREAD

**Issue by pjnagel** [hashicorp/packer#7679]
Packer filters out empty newlines from the output of the shell-local provisioner (and maybe others too?).

If the command you are running has a lot of output, split by newlines into different logical chunks, then the output becomes very hard to parse visually, which makes debugging the provisioner step difficult. It's not just that the output is all squished together, but also that the output is _different_ from one is used to for the particular tool being used.

For example here is some output as expected:
```
PLAY [Setup and maintain Tau teamcity] *****************************************

TASK [Gathering Facts] *********************************************************
Wednesday 22 May 2019  13:13:19 +0000 (0:00:00.434)       0:00:00.435 ********* 
ok: [lxd1.lautus.net]

TASK [Ensure hosts with aliases use wildcard certificate] **********************
Wednesday 22 May 2019  13:13:21 +0000 (0:00:02.431)       0:00:02.866 ********* 

TASK [Include encrypted vault variables] ***************************************
Wednesday 22 May 2019  13:13:21 +0000 (0:00:00.047)       0:00:02.914 ********* 
ok: [lxd1.lautus.net]

TASK [core_mechanics/config : Check if host is on expected OS] *****************
Wednesday 22 May 2019  13:13:21 +0000 (0:00:00.156)       0:00:03.070 ********* 
```

And here is the equivalent output of the same tool run under packer:
```
tau-container: PLAY [Setup and maintain Tau teamcity] *****************************************
tau-container: TASK [Gathering Facts] *********************************************************
tau-container: Wednesday 22 May 2019  13:02:12 +0000 (0:00:00.443)       0:00:00.443 *********
tau-container: ok: [packer-tau-container.lautus.net]
tau-container: TASK [Ensure hosts with aliases use wildcard certificate] **********************
tau-container: Wednesday 22 May 2019  13:02:17 +0000 (0:00:05.201)       0:00:05.644 *********
tau-container: TASK [Include encrypted vault variables] ***************************************
tau-container: Wednesday 22 May 2019  13:02:17 +0000 (0:00:00.047)       0:00:05.692 *********
tau-container: ok: [packer-tau-container.lautus.net]
tau-container: TASK [core_mechanics/config : Check if host is on expected OS] *****************
tau-container: Wednesday 22 May 2019  13:02:17 +0000 (0:00:00.155)       0:00:05.848 *********
```

- Packer version 1.4.0.
- Host platform Ubuntu 16.04.

**Comment by SwampDragons** [comment]
Hi there! I was able to reproduce this issue on v1.4.0, but can't reproduce on the master branch of Packer or on v1.4.1, so I think it's been fixed. I'm closing for now, but if you upgrade and find the problem still exists for you, we can reopen.
